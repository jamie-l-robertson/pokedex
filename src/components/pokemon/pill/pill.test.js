import React from 'react';
import { Pill } from './';

const Type = mount(<Pill label="Grass" />);

describe('<Pill />', () => {
  it('Pill renders', () => {
    expect(Type).toMatchSnapshot();
  });

  it('Pill has the correct value', () => {
    expect(Type.find('li').text()).toEqual('Grass');
  });
});
