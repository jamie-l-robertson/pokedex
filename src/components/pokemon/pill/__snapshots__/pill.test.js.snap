// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Pill /> Pill renders 1`] = `
<styled.li
  className="pill-grass"
>
  <li
    className="pill-grass sc-bdVaJa kdfYYr"
  >
    Grass
  </li>
</styled.li>
`;
